src/foam/parse/FObjectArrayParser.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.parse;
+
+import foam.lib.json.Whitespace;
+import foam.lib.parse.*;
+
+public class FObjectArrayParser extends ProxyParser {
```

Comment:
Again this looks the same as the parser in foam/lib/json/FObjectArrayParser.java let's just use that if we can.